File: sem7/src/main/java/ru/gb/oseminar/service/HighService.java
package ru.gb.oseminar.service;

import ru.gb.oseminar.interfaces.ISmoke;
import ru.gb.oseminar.interfaces.IStudy;

public class HighService implements ISmoke, IStudy{

    @Override
    public void study() {
        System.out.println("Я должен учиться");
    }

    @Override
    public void smoke() {
        System.out.println("Я бросаю курить");
    }
    
    public void prepareToExam() {
        System.out.println("Я готовлюсь к экзаменам");
    }

    public void getLove() {
        System.out.println("У меня отношения!");
    }
}

File: FinalHW-OOP/service/HighService.cs
namespace FinalHWOOP {
    public class HighService : ISmoke, IStudy {
        public void smoke() {
            Console.WriteLine("Я бросаю курить");
        }

        public void study() {
            Console.WriteLine("Мне надо учиться");
        }

        public void getLove() {
            Console.WriteLine("У меня есть отношения");
        } 

        public void prepareToExam() {
            Console.WriteLine("Я готовлюсь к экзаменам");
        }
    }
}
